.github/workflows/ghcr.yml

```
@@ -0,0 +1,38 @@
+# Mirror DockerHub images used by the Rust project to ghcr.io.
+#
+# In some CI jobs, we pull images from ghcr.io instead of Docker Hub because
+# Docker Hub has a rate limit, while ghcr.io doesn't.
+# Those images are pushed to ghcr.io by this job.
+#
+# Note that authenticating to DockerHub or other registries isn't possible
+# for PR jobs, because forks can't access secrets.
+# That's why we use ghcr.io: it has no rate limit and doesn't require authentication.
+
+name: GHCR
```

```suggestion
name: GHCR image mirroring
```
or something like that, to make it more obvious.